test/clj_meowallet/core_test.clj

```diff
@@ -73,3 +74,22 @@
 
   (testing "success"
     (is (result/failed? result)))))
+
+(deftest verify-callback-test
+  (if-not (env :meo-wallet-api-key)
+    (println "Warning: No meo wallet api key on env (ignoring test)")
+
+    (let [credentials {:meo-wallet-api-key (env :meo-wallet-api-key)}
+          data {:amount 10
+                :currency "EUR"
+                :event "COMPLETED"
+                :ext-customerid "00001"
+                :ext-email "noreply@sapo.pt"
+                :ext-invoiceid "38440200100"
+                :method "WALLET"
+                :operation-id "qwkjehqkjwhe"
+                :operation-status "COMPLETED"
+                :user "237"}
+        result (<!! (core/verify-callback credentials data))]
+
+    (is (not (result/succeeded? result))))))
```

Comment:
nobody generated that mb-ref with that operation-id. so I'm expecting a failed result